t/ui/14-dashboard-parents.t

```diff
@@ -84,6 +84,18 @@ $driver->find_element_by_link_text('Build0091')->click();
 my $element = $driver->find_element_by_link_text('opensuse');
 ok($element->is_displayed(), 'link to child group expanded');
 $driver->find_element_by_link_text('Build0091')->click();
+
+subtest 'progress bar link' => sub {
+    my $first_build_row = $driver->find_elements('.build-row')->[0];
+    my $link = $driver->find_child_element($first_build_row, '.progress-bar-failed a');
+    is($link->get_text, '1 failed', 'link text');
+    like(
+        $link->get_attribute('href'),
+        qr|.*/tests/overview\?result=failed&distri=opensuse&version=Factory&build=87\.5011&groupid=1001&groupid=1002|,
+        'link href'
+    );
```

Comment:
I would prefer that to be a bit more concise. Also we don't need to check that much, just look for the result and multiple group ids, like so

```suggestion
    like $link->get_attribute('href'), qr|.*/tests/overview\?result=failed&.*&groupid=1001&groupid=1002|, 'link href';
```

Reply:
To save a few lines:
```suggestion
    my $r = qr|.*/tests/overview\?result=failed&distri=opensuse&version=Factory&build=87\.5011&groupid=1001&groupid=1002|;
    like($link->get_attribute('href'), $r, 'link href');
```

Reply:
Ok, then use the suggestion from @Martchus